app/models/series.rb

```diff
@@ -227,14 +228,14 @@ def Series.load_all_mean_corrected_sa_series_from(spreadsheet_path, sheet_to_loa
   
   def Series.load_all_series_from(spreadsheet_path, sheet_to_load = nil, priority = 100)
     t = Time.now
-    # puts "Setting priority to #{priority}"
     each_spreadsheet_header(spreadsheet_path, sheet_to_load, false) do |series_name, update_spreadsheet|
-      @data_source = Series.store(series_name, Series.new(:frequency => update_spreadsheet.frequency, :data => update_spreadsheet.series(series_name)), spreadsheet_path, %Q^"#{series_name}".tsn.load_from "#{spreadsheet_path}", "#{sheet_to_load}"^) unless sheet_to_load.nil?
-      @data_source = Series.store(series_name, Series.new(:frequency => update_spreadsheet.frequency, :data => update_spreadsheet.series(series_name)), spreadsheet_path, %Q^"#{series_name}".tsn.load_from "#{spreadsheet_path}"^) if sheet_to_load.nil?      
-      #puts series_name
+      eval_format = sheet_to_load ? '"%s".tsn.load_from "%s", "%s"' : '"%s".tsn.load_from "%s"'
+      @data_source = Series.store(series_name,
+                                  Series.new(frequency: update_spreadsheet.frequency, data: update_spreadsheet.series(series_name)),
+                                  spreadsheet_path,
+                                  eval_format % [series_name, spreadsheet_path, sheet_to_load])
+
```

Comment:
Note that the above changed lines (232-37) are just a refactor for readability, because I was so offended when I saw it :=}